### PR TITLE
Update advanced settings in index.md

### PR DESCRIPTION
**Pull request by ckhordiasma**
As of 24.10.0, simplefin is not in experimental features anymore, and now there is a customizable reports option.

### CODE REVIEW

File: docs/settings/index.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ If you are experiencing bugs relating to split transactions and the “Reset bud
 
 Be careful with this section; you might encounter irrecoverable errors if you enable these settings. But if you want to test out some bleeding-edge features, here's your place to do it. We _highly_ recommend backing up your budget before continuing, just in case. As of the [latest release](../releases.md), the current available experimental features are:
 
-- Monthly spending
-- Budget mode toggle
-- Goal templates
-- SimpleFIN sync
+- [Monthly spending](../experimental/monthly-cleanup.md)
```
Comment (youngcw) [inline docs/settings/index.md:55]: this doc has nothing to do with the monthly spending report

File: docs/settings/index.md [hunk 2]
```diff
@@ -52,7 +52,7 @@ If you are experiencing bugs relating to split transactions and the “Reset bud
...
+- [Budget mode toggle](../experimental/tracking-budget.md)
+- [Goal templates](../experimental/goal-templates.md)
+- Customizable reports
```
Comment (youngcw) [inline docs/settings/index.md:58]: should specify that its customizable dashboards.  Custom reports are already mainlined

File: docs/settings/index.md (hunk 1 above)
Comment (ckhordiasma) [reply docs/settings/index.md:55]: is there a docs page that covers the monthly spending report?

File: docs/settings/index.md (hunk 2 above)
Comment (ckhordiasma) [reply docs/settings/index.md]: ok, I will reword to exactly as it appears in the settings page

**Review by youngcw — APPROVED** [review]

**Pull request merged**